src/util/heap/layout/map64.rs

```
@@ -152,6 +152,16 @@ impl Map for Map64 {
         unreachable!()
     }
 
+    fn get_available_discontiguous_chunks(&self) -> usize {
+        // We don't use discontiguous chunks for 64-bit
+        0
+    }
+
+    fn get_chunk_consumer_count(&self) -> usize {
+        // We don't use discontiguous chunks for 64-bit
+        0
```

This should panic instead of returning 0.